README.md

```diff
@@ -10,6 +10,12 @@
 This code that powers the UN Office for the Coordination of Humanitarian Affairs (UN-OCHA)
 Taxnonomy as a Service APIs.
 
+## OSX requirements
```

Comment:
And.... these aren't are unique to OSX either. `easy_install` and `pip` should be the same on all Python systems. :)